Core/DISET/private/Transports/SSL/SocketInfoFactory.py

```diff
@@ -57,7 +57,7 @@ def __sockConnect( self, hostAddress, sockType, timeout, retries ):
     osSocket = socket.socket( sockType, socket.SOCK_STREAM )
     #osSocket.setblocking( 0 )
     if timeout:
-      osSocket.settimeout( 5 )
+      osSocket.settimeout( 0.5 ) # we retry 3 times, which is 3 sec + 5 times retry = 15 sec.
```

Comment:
I am going to test the Bookkeeping service and I will generate high workload. So, I can see how this settings will behaves.